src/util/number.ts

```diff
@@ -275,8 +292,7 @@ export function getPercentWithPrecision(valueList: number[], idx: number, precis
         remainder[maxId] = 0;
         ++currentSum;
     }
-
-    return seats[idx] / digits;
+    return seats;
```

Comment:
Here I think it's better to map the seats to a new array in which each seat is divided by the `digits`. Note that `zrUtil.map` is preferred.